### PR TITLE
Added form & POST request

### PR DESCRIPTION
**Pull request by kristen-m**
Accidentally have 2 commits claiming to add the form element because I forgot where I left off. In the second one I added the comment container.

### CODE REVIEW

**Review by juliareichel — COMMENTED** [review]
I would delete the function getFruit() which is commented out for cleaner code. Other than that, looks good to me!

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -23,23 +23,40 @@
...
+  }
+
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      String comment = getComment(request);
+      comments.add(comment);
+      response.setContentType("text/html");
+      response.sendRedirect("/index.html");
+  }
+
+  /**
```
Comment (damargulis) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: If you're going to include a javadoc for a function (unlike in javascript, javadocs are not required for private functions, but are always nice to include if they help the codes readability), make sure you include all of the applicable tags (@param, @return, @throws and/or @deprecated).  In this case, make sure you add a param tag for the request variable.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -23,23 +23,40 @@
...
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Gson gson = new Gson();
-    String jsonfruit = gson.toJson(fruits);
     response.setContentType("application/json;");
-    response.getWriter().println(jsonfruit);
+    String json = new Gson().toJson(comments);
+    response.getWriter().println(json);
+  }
+
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```
Comment (damargulis) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:42]: I think this should have an @Override tag?

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -119,6 +115,18 @@ <h2>Fun Facts</h2>
                 <li>Cutting down on my screentime</li>
                 <li>Going on hikes around town</li>
             </ul>
+
+            <form action="/data" method="POST">
+
+              <p>Have something to say? Let me know:</p>
+              <textarea name="text-input">Post your comment here!</textarea>
+              <br/><br/>
```
Comment (damargulis) [inline portfolio/src/main/webapp/index.html]: Avoid br tags (especially several in a row).
Wrap items in divs or other elements if you need to create linebreaks, and use css if you need to position things with greater spacing

**Review by damargulis — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (kristen-m) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:42]: It definitely should! Thanks for catching that

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (kristen-m) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Noted! Thanks for including those other tags, I was only aware of @return and @param

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (kristen-m) [reply portfolio/src/main/webapp/index.html]: Fixed!

**Pull request merged**